docs/config_extra.md

```diff
@@ -0,0 +1,48 @@
+# Additional Device Specific Configuration Settings
+These settings can be set using the following format:
+```yaml
+insteon:
+  devices:
+    switch:
+      - aa.bb.cc: lamp1
+        extra_config: 1
+      - 11.22.33: lamp2
+```
+In this case, `lamp1` would have the value of 1 set for the `extra_config` setting.  This setting __only__ affects `lamp1` and will not affect `lamp2`.
+
+> Device names __must be defined__ to use a device specific configuration.
+
+> Note the indentation and __the lack of a dash__ on the line specifying the device specific configuration entry.
+
+## Settings for Specific Device Types
+Some device types have additional settings.
+
+> No such settings yet.
+
+## Advanced Settings
+These settings can be used with all device types, but should be considered advanced.  These settings may cause undesirable effects.
+
+### `min_hops` - Minimum Hops
+If a specific device frequently takes between 5-15 seconds to respond to a command.  Try setting the `min_hops` for that device to 1.  Restart InsteonMQTT and test out the device.  If there continues to be a delay, `min_hops` can be increased to 2 or 3.  Ideally, you want __the lowest value for `min_hops` that removes the delay.__
+
+For example, the configuration of a device in `config.yaml` could look like:
+```yaml
+insteon:
+  devices:
+    switch:
+      - aa.bb.cc: lamp1
+        min_hops: 1
+      - 11.22.33: lamp2
+```
+In this case, all messages sent to `lamp1` would have a minimum of 1 hops set.  While `lamp2` would continue to use the calculated smart-hops value.
+
+> Setting `min_hops` likely will __not help__ a device that frequently fails to respond to commands at all.
+
+#### Detailed Minimum Hops Explanation
+Each message sent _to_ a device has a hops value (between 0 and 3) that defines the number of times the message will be rebroadcast to the network.  InsteonMQTT calculates the optimum hop count by monitoring the number of hops it takes for messages _from_ the device to reach the Modem.
```

Comment:
Sure if you want.  It should be a warning not an error.  I doubt anyone would see it in all the startup messages.  If you push a commit I will merge it.

Reply:
You make a good point about a warning going unnoticed during start up.  Would it be better to log the warning every time the too-high min_hops value is used with a device (i.e. every time a message is sent to the device), or would that be way too verbose?

Reply:
I see this as a non-issue, the documentation is clear that the max hops is three.  If a user sets a value higher than 3, it just uses three.  A warning on startup isn't a big deal, but I doubt it will do much.  A constant warning seems overkill for something that isn't really an issue.

Reply:
Sounds good.  Let's leave it alone, then.  It's an advanced feature, after-all.